sources/academy/tutorials/apify_scrapers/getting_started.md

```diff
@@ -152,7 +152,7 @@ In the structures, only the `OWNER` and `NAME` change. We can leverage this in a
 
 #### Making a pseudo URL
 
-**Pseudo URL**s are really just URLs with some variable parts in them. Those variable parts are represented by [regular expressions](https://regexone.com/) enclosed in brackets `[]`.
+**Pseudo URL**s are really just URLs with some variable parts in them. Those variable parts are represented by [regular expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) enclosed in brackets `[]`.
```

Comment:
This has nothing to do with this particular PR, but I think that this section is outdated and the preferred way of doing things are now globs, not pseudo URLs.

Reply:
Thanks for the suggestion, filed as https://github.com/apify/apify-docs/issues/1009